StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/v1-elements-sessions/CustomerSession.swift

```diff
@@ -78,6 +85,7 @@ struct PaymentSheetComponent: Equatable, Hashable {
 struct PaymentSheetComponentFeature: Equatable, Hashable {
     let paymentMethodSave: Bool
     let paymentMethodRemove: Bool
+    let paymentMethodSaveAllowRedisplayOverride: STPPaymentMethodAllowRedisplay?
```

Comment:
Can we document this somehow? Maybe just have a docstring for `PaymentSheetComponentFeature ` that links to the api https://docs.corp.stripe.com/api/customer_sessions/object#customer_session_object-components-payment_sheet